src/views/v2/event-status/single/event-statuses/postponed-status.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Status for a Postponed Event.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/single/event-statuses/postponed-status.php
```

Comment:
this and the file path should be `events-status/single/event-statuses-container/canceled-status.php`, seeing as you changed the including template name